index.html

```diff
@@ -76,8 +84,8 @@ <h6 class="second_subtitle_card">2015</h6>
                 </div>
             </div>
 
-            <div class="main_section">
-                <div>
+            <div class="section-2 main_section">
+                <div class="content_image_card">
```

Comment:
- [optional] great job making the requested changes. please reduce the gap between your cards so the match the figma design
yours
![Screenshot (918)_LI](https://user-images.githubusercontent.com/65298048/228965873-cde5f678-8826-4331-8ac7-06f3d8e8f89d.jpg)
figma
![Screenshot (917)](https://user-images.githubusercontent.com/65298048/228965918-d9cfacec-3135-4823-affb-76100fea6bfe.png)